ts/util/EncryptionUtil.ts

```diff
@@ -0,0 +1,42 @@
+import * as crypto from 'crypto';
+
+class EncryptionUtil {
+
+    public static encrypt(secret) {
+        const cet = process.env.CET || EncryptionUtil.defaultKey;
+        const cipher = crypto.createCipher('aes256', cet.startsWith('oCB') ? EncryptionUtil.defs(cet, 13) : cet);
+        let encrypted = cipher.update(secret, 'utf8', 'hex');
+        encrypted += cipher.final('hex');
+        return encrypted;
+    }
+
+    public static decrypt(secret) {
+        const cet = process.env.CET || EncryptionUtil.defaultKey;
+        const decipher = crypto.createDecipher('aes256', cet.startsWith('oCB') ? EncryptionUtil.defs(cet, 13) : cet);
+        let decrypted = decipher.update(secret, 'hex', 'utf8');
+        decrypted += decipher.final('utf8');
+        return decrypted;
+    }
+
+    private static defaultKey = 'oCB@qs?nDn@E=nsC?s?q@CsFDq=EsEDoD>rBBqEor=@Anop>rAoDBEop>s@s=qrE';
+    // ref: https://gist.github.com/olastor/54c78a3d29c69806c57a32eff32f191a
+    private static obfs(str, key) {
```

Comment:
I like the encryption (creds aren't stored in plain text on the file system), but I don't think this adds much to the encryption, just adds some dev complexity. I'd opt for keeping the encryption and just use a hard-coded encryption key.

Either way, it is not required to store your password in the config. Every command has the option to pass in the password at runtime. We could update the docs to be transparent about the convenience vs security risk.

Reply:
I've been on the fence but I think I agree.  I'll remove the obfuscation.  We can revisit if it becomes an issue and update docs as you suggest.